### PR TITLE
Add job description

### PR DESCRIPTION
**Pull request by michaelcarter**

### CODE REVIEW

File: src/pages/careers/front-end-developer.pug [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            li.styled-list__item You have experience with another programming language.
+
+          h4 Location
+          p This is a remote position. However, when we're not quarantined, a few times a year we like everyone to drop into Canterbury - famous for its cathedrals, cobbles and tales - for a bit of face-time with the rest of the team.
+
+          h4 How to apply
+          p Email your CV to <a href="mailto:hello@weareleaf.com">hello@weareleaf.com</a>, along with a little about yourself, some examples of your work, and why you think you'd be a good fit.
+
+          h4 After you apply
+          p If we're interested in moving forwards with your application, we'll arrange a remote interview with you to further establish our suitability for you as employers, and your suitability for this role.
```
Comment (tinavanschelt) [inline src/pages/careers/front-end-developer.pug:75]: *forward

File: src/pages/careers/front-end-developer.pug [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+          +summary({
+            summaryHeading: "Who are we?",
+            summaryText: "Leaf are a remote agency based in the UK. We specialise in complex information capture, helping organisations excel online by making it easy for their users to provide them with high quality information."
+          })
+
+        .grid__item
+          h4 About the role
+          p We're working hard to build a strong core of expertise around complex information capture on the web. To get there, we need help from a brilliant front-end developer who loves to build fast, accessible, progressively enhanced web applications that deliver tremendous value.
+
+          p In this role you'll be working remotely and choosing your own hours, but within that flexibility you'll be expected to deliver features and improvements in an efficient, timely manner. We can't promise every day will be radically different and exciting, but we also know not everyone wants that. You'll spend most of your time problem solving, writing code, and shipping value.
```
Comment (chrisannetts) [inline src/pages/careers/front-end-developer.pug:36]: Putting myself in job-hunting shoes, the last two sentences are a real-turn off for me. Could we flip the negative and outline who it's ideal for instead?

> The role is ideal for someone who likes stability and routine, as you'll spend most of your time problem solving, writing code, and shipping value.

File: src/pages/careers/front-end-developer.pug [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+      figure
+        img(src="/assets/images/careers/image-2.jpg" alt="Chris and Mike on a face-to-face team catch-up day")
+        figcaption Chris and Mike on a face-to-face team catch-up day
+
+      .grid.grid--two-thirds
+        .grid__item
+          h4 Primary responsibilities
+          ul.styled-list.styled-list--inside-lg
+            li.styled-list__item Efficient delivery of high quality features and improvements to our client's web applications.
```
Comment (chrisannetts) [inline src/pages/careers/front-end-developer.pug:46]: I'm probably wrong, but should this be "clients"?

File: src/pages/careers/front-end-developer.pug [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+            li.styled-list__item Driving the technical quality of our work, and knowledge around complex data capture forward.
+
+          h4 You will need
+          ul.styled-list.styled-list--inside-lg
+            li.styled-list__item Excellent written and verbal communication skills.
+            li.styled-list__item An excellent understanding of modern HTML, CSS, and JavaScript.
+            li.styled-list__item A good high level understanding how web browsers work (HTTP, the DOM, etc.).
+            li.styled-list__item A good understanding of how to use your browser's development tools.
+            li.styled-list__item A good understanding of issues related to user experience, performance, accessibility, progressive enhancement, and cross-browser compatibility.
+            li.styled-list__item A willingness to learn new technologies, and programming languages.
```
Comment (chrisannetts) [inline src/pages/careers/front-end-developer.pug:58]: I don't think the comma is needed after 'technologies'.

**Pull request merged**